test/e2e/build_test.go

```diff
@@ -786,17 +786,18 @@ RUN ls /dev/test1`, ALPINE)
 
 	It("podman build use absolute path even if given relative", func() {
 		containerFile := fmt.Sprintf(`FROM %s`, ALPINE)
-		err = os.Mkdir("relative", 0755)
+		relativeDir := filepath.Join(podmanTest.TempDir, "relativeDir")
+		containerFilePath := filepath.Join(relativeDir, "Containerfile")
+		buildRoot := filepath.Join(relativeDir, "build-root")
+
+		err = os.Mkdir(relativeDir, 0755)
 		Expect(err).To(BeNil())
-		containerFilePath := filepath.Join("relative", "Containerfile")
-		err = os.Mkdir("relative/build-root", 0755)
+		err = os.Mkdir(buildRoot, 0755)
 		Expect(err).To(BeNil())
 		err = ioutil.WriteFile(containerFilePath, []byte(containerFile), 0755)
 		Expect(err).To(BeNil())
-		build := podmanTest.Podman([]string{"build", "-f", "./relative/Containerfile", "./relative/build-root"})
+		build := podmanTest.Podman([]string{"build", "-f", containerFilePath, buildRoot})
 		build.WaitWithDefaultTimeout()
 		Expect(build).To(Exit(0))
-		err = os.RemoveAll("relative")
-		Expect(err).To(BeNil())
```

Comment:
shouldn't this be kept, or converted to use `relativeDir`?

Reply:
I think this is ok to rm since he put all of the new files in `podmanTest.TempDir`